src/store/index.tsx

```diff
@@ -2,22 +2,11 @@ import React, { useState, useEffect } from 'react'
 import { useWebsiteStatus } from 'components/hooks/use-website-status'
 import { isEuCountry, isP2PAllowedCountry, isUK } from 'common/country-base'
 
-type ProviderType = {
-    is_eu_country?: string
-    is_p2p_allowed_country?: string
-    crypto_config?: string
-    website_status?: string
-    website_status_loading?: string
-    setWebsiteStatus?: string
-    user_country?: string
-}
-
 type DerivProviderProps = {
-    value?: ProviderType
```

Comment:
Any reason you removed this?

Reply:
> Any reason you removed this?

I decided to keep store/index.tsx unchanged as much as possible to prevent conflicts. Should I restore it?